### DISCUSSION TITLE
关于开屏广告以外的规则默认禁用的通知

### DISCUSSION THREAD

**Discussion by lisonge** [gkd-kit/gkd#355] [General]
随着默认规则数量的增多，仅凭 开发者的主观判断 来决定规则的默认启用 已不太现实

因此默认规则已经变更禁用逻辑，除开屏广告外其它所有规则的默认值将被设置为禁用

您(用户) 需要自行手动选择需要开启的规则

---

此外下个版本将新增 `规则分类功能` 以供用户手动批量打开/关闭规则(某些特定的规则如青少年弹窗/更新弹窗/通知权限弹窗等)

**Comment by lisonge** [comment]
手动 at 所有贡献者，如果这条消息打扰到您，请您 忽略/评论/邮箱 告知我关闭对您的 at

@Snoopy1866 @lisonge @jiuqianyuan @github-actions[bot] @walklty @aoguai @xiaowonet @adproqwq @CMBill @yanyu2023 @dlh996 @Otis0227 @shanlanCoding @BreezyOrigin @Sec-ant @zeXtreme @superHao2000 @longm21 @xe5700 @JammyLin @Nebulafalls @OldCoding @Ran0929 @YaChengMu @shjdgwj @zhyx490991014 @zhangyusheng211 @lliioollcn @lyc452 @M1saka10010 @xuanqb @sishuikaki @sherylynn @ZhReimu @bleach2012 @javabcde @kyrinyoga @zhuzhuani @wjf0214 @uuiid @MoeGrid @g1fts @small-ora @China-Uncle @doio @Crty @someone120 @koo6666 @h7221836 @maxchang3 @Schweik7 @weirongchuan @pengxinwen @Zonezzc @alzuobaba @guanyintu @marryton007 @foxlesbiao @lsxuan12138 @luoyan87 @ppengit @w4616 @shiquda @tiygu @xhemj @xzhlksh @yakun4566 @yangqiwen1024 @ylxz95 @ysjsgzq @zhcode-fun @yunyuyuan

**Comment by jiuqianyuan** [comment]
您好，看到您已修改默认订阅发布规则[feat: 仅启用开屏广告](https://github.com/gkd-kit/subscription/commit/303f1f8f3836507aefafed59e107cbf08ff5f4ce)。
那我们以后提交PR时，需要直接将非开屏规则设置为`enable: false,`吗，还是暂不作要求，可先继续保留主观启用建议

**Reply by lisonge** [reply]
不需要设置 enable 这个字段

apps/*.ts 下的规则被转换为 json 时会将非开屏广告的 enable 被强制设置为 false

<https://github.com/gkd-kit/subscription/commit/303f1f8f3836507aefafed59e107cbf08ff5f4ce#diff-c3095d5010e65c52737a98a5d618ea24049ebe90c8470752426081d70ed6e012R22-R26>

**Reply by jiuqianyuan** [reply]
👌
